pkg/operator/controllers/workaround/ifreload.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/Azure/ARO-RP/pkg/util/version"
 )
 
-const kubeNamespace = "openshift-azure-ifreload"
+const kubeNamespaceIfReload = "openshift-azure-ifreload"
```

Comment:
Didn't we talked in the scrum that we need to drop the workaround controller and just start writing separate controllers for each component/fix? Bit of code duplications, but easier to maintain and less customer naming logic. 

Or you had some other discussions after this? @troy0820 @jim-minter  ?

Reply:
I think because the ifReload is going away and the system reserved doesn't have a specific date we were using this to have routefix be managed by the operator.  I agree having a controller for each piece is a better idea for maintaining.  We can talk about this again if this isn't the path forward.

Reply:
VSTS created to overall refactor of this package